Source/WebCore/rendering/RenderBlockFlow.cpp

```diff
@@ -678,7 +678,10 @@ inline LayoutUnit RenderBlockFlow::shiftForAlignContent(LayoutUnit intrinsicLogi
     // Calculate alignment shift.
     LayoutUnit computedLogicalHeight = logicalHeight();
     LayoutUnit space = computedLogicalHeight - intrinsicLogicalHeight;
-    if (space <= 0 && OverflowAlignment::Unsafe != alignment.overflow())
+    if (space <= 0
+        && (OverflowAlignment::Safe == alignment.overflow()
```

Comment:
We don't generally put the values on the LHS of an equality check (which people do to avoid assignment in the case of a typo), so I'd flip these around to match other code.

Reply:
Also, with these more complex boolean conditions, it often helps readability to use a lambda function with a helpful name, which I think would improve things in this case. It's hard to tell what this condition is trying to do.

Reply:
Would something like this work?
```
    if (space <= 0) {
        bool overflowIsSafe = alignment.overflow() == OverflowAlignment::Default && !isScrollContainerY()
            || alignment.overflow() == OverflowAlignment::Safe
            || alignment.position() == ContentPosition::Normal;
        if (overflowIsSafe)
            return 0_lu; // Floored at zero; we're done
    }
```

It's a temp variable instead of a lambda, but I think that seems simpler than a lambda.